tests/fixtures/json_content_99999.py

```diff
@@ -0,0 +1,87 @@
+# coding=utf-8
+
+from collections import OrderedDict
+
+expected = OrderedDict([
+    ('id', u'99999'),
+    ('title', u'Fishing for errors in the\xa0tests'),
```

Comment:
weird character, the non-breaking space? Where did it come from, the parsing produced it and was reported here?

Reply:
This character was in an original ``.docx`` sample I was provided, and I didn't change it, this being a result of real data unaltered if it flows through.

Reply:
There may be a character encoding problem here, doing more checks